### DISCUSSION TITLE
[SOLVED] Can i enable notification for only few feeds?

### DISCUSSION THREAD

**Discussion by ghost** [martinrotter/rssguard#1124] [Q&A]
There is only a per feed option to disable notifications, is there a way to somehow do the opposite? only enable notification for few feeds. thanks

**Comment by inson1** [comment]
@martinrotter ?

**Reply by martinrotter** [reply]
well, kind of yes.

1. right click account node -> edit child feeds (recursive) -> disable notifications for them in dialog that show up
2. individually select "new" feeds or add new feed -> "edit selected items" -> enable notifications in the dialog
